### PR TITLE
Homework 6 is done

### PR DESCRIPTION
**Pull request by LizRumyantseva**
JDI homework is ready

### CODE REVIEW

File: src/main/java/hw6/entities/Data.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package hw6.entities;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Data {
```
Comment (troe88) [inline src/main/java/hw6/entities/Data.java]: What kind of data is this ? You need to named class with referense to it purpose.

File: src/main/java/hw6/entities/MetalsAndColorsData.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package hw6.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MetalsAndColorsData {
+    private Data data_1;
```
Comment (troe88) [inline src/main/java/hw6/entities/MetalsAndColorsData.java]: This is completely prohibited to read the file in according to amount of data. Just imagine the situation if you need to add one more rows. What will happen in this case ?

File: src/main/java/hw6/forms/MetalsAndColorsForm.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    private void selectElements(Data data) {
+        for (String element : data.getElements()) {
+            checklistElements.select(element);
+        }
+    }
+
+    private void selectSummaryOddRadio(Data data) {
+        oddRadio.select(String.valueOf(data.getSummaryOdd()));
+
```
Comment (troe88) [inline src/main/java/hw6/forms/MetalsAndColorsForm.java]: Empty line should not be at the end of the methods..

File: src/test/java/hw5/Hooks/CucumberHooks.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package hw5.Hooks;
```
Comment (troe88) [inline src/test/java/hw5/Hooks/CucumberHooks.java]: Basically, this files have no relation to HW-6, so they should not be in pull request

**Review by troe88 — CHANGES_REQUESTED** [review]

File: src/main/java/hw6/entities/MetalsAndColorsData.java (hunk 2 above)
Comment (LizRumyantseva) [reply src/main/java/hw6/entities/MetalsAndColorsData.java]: Yes, it's wrong. Now I have read about TypeToken

File: src/main/java/hw6/forms/MetalsAndColorsForm.java (hunk 3 above)
Comment (LizRumyantseva) [reply src/main/java/hw6/forms/MetalsAndColorsForm.java]: Fixed

File: src/main/java/hw6/entities/Data.java (hunk 1 above)
Comment (LizRumyantseva) [reply src/main/java/hw6/entities/Data.java]: Renamed

File: src/test/java/hw5/Hooks/CucumberHooks.java (hunk 4 above)
Comment (LizRumyantseva) [reply src/test/java/hw5/Hooks/CucumberHooks.java]: They are from previous pull request...(a day apart) Should I do something with this now when they are merged?

**Comment by LizRumyantseva** [comment]
I have rebased hw6 branch and pushed it again

File: src/test/java/hw5/Hooks/CucumberHooks.java (hunk 4 above)
Comment (LizRumyantseva) [reply src/test/java/hw5/Hooks/CucumberHooks.java]: rebased

**Review by troe88 — APPROVED** [review]

**Pull request merged**